<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:rgb(52,51,62);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_File_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border:2px solid rgb(255,255,255)
color:rgb(255,255,255)

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>File name</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Audio_File_Duration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal{
	height:10px;
	width:320px;
	background:red;
	border-radius:5px;
}
QSlider::handle:horizontal{
	background:rgb(255,255,255);
	width:14px;
	height:14px;
	margin:-5px -5px;
	border-radius:7px;

}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Audio_Volume">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal{
	height:10px;
	width:250px;
	background:red;
	border-radius:5px;
}
QSlider::handle:horizontal{
	background:rgb(255,255,255);
	width:14px;
	height:14px;
	margin:-5px -5px;
	border-radius:7px;

}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Mute">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:20px;
	background-color:rgb(255,255,255);
	color:rgb(52,51,64);
	padding: 0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_Value_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(255,255,255)
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Value_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(255,255,255)
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>341</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Back">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:none;
	border-radius:30px;
	background-color:rgb(255,255,255);
	color:rgb(52,51,64);
	min-width:60px;
	max-width:60px;	
	min-height:60px;
	max-height:60px;
	padding: 0px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Stop">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:none;
	border-radius:30px;
	background-color:rgb(255,255,255);
	color:rgb(52,51,64);
	min-width:60px;
	max-width:60px;	
	min-height:60px;
	max-height:60px;
	padding: 0px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Play">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:none;
	border-radius:30px;
	background-color:rgb(255,255,255);
	color:rgb(52,51,64);
	min-width:60px;
	max-width:60px;	
	min-height:60px;
	max-height:60px;
	padding: 0px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Pause">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:none;
	border-radius:30px;
	background-color:rgb(255,255,255);
	color:rgb(52,51,64);
	min-width:60px;
	max-width:60px;	
	min-height:60px;
	max-height:60px;
	padding: 0px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Forward">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:none;
	border-radius:30px;
	background-color:rgb(255,255,255);
	color:rgb(52,51,64);
	min-width:60px;
	max-width:60px;	
	min-height:60px;
	max-height:60px;
	padding: 0px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>320</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:30px;
	background-color:rgb(255,255,255);
	color:rgb(52,51,64);
	min-width:60px;
	max-width:60px;	
	min-height:60px;
	max-height:60px;
	padding: 0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>382</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_file"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
